CMakeLists.txt

```diff
@@ -194,6 +243,40 @@ if(${TRITON_ENABLE_ONNXRUNTIME_OPENVINO})
   )
 endif() # TRITON_ENABLE_ONNXRUNTIME_OPENVINO
 
+#
+# ONNX Runtime - if requested, build ORT using docker
```

Comment:
Re-word to "Build ONNX Runtime using docker" for clarity. This can be interpreted as builds ORT using docker if ORT is requested.